census/models.py

```diff
@@ -15,7 +15,7 @@
 class Event(models.Model):
     title = models.CharField(max_length=100, help_text="Title or short description of the event")
     description = models.TextField(blank=True, help_text="Full description of the event")
-    recurrences = RecurrenceField(default=None, blank=True, help_text="Enter date of the event or information on a recurring event")
+    date_information  = RecurrenceField(default=None, blank=True, help_text="Add date of the event or rules for a recurring event")
```

Comment:
Instead of renaming the field, you can just specify a `labels` dict in the [ModelForm](https://docs.djangoproject.com/en/2.2/topics/forms/modelforms/#overriding-the-default-fields). I think that would be better.